hardware/cmdb.py

```diff
@@ -28,48 +28,50 @@ class CmdbError(Exception):
 
 
 def cmdb_filename(cfg_dir, name):
-    'Return the cmdb filename.'
+    """Return the cmdb filename."""
     return os.path.join(cfg_dir, name + '.cmdb')
 
 
 def load_cmdb(cfg_dir, name):
-    'Load the cmdb.'
+    """Load the cmdb."""
     filename = cmdb_filename(cfg_dir, name)
     try:
         return eval(open(filename).read(-1))
-    except IOError as xcpt:
+    except OSError as xcpt:
         if xcpt.errno != errno.ENOENT:
```

Comment:
I suspect there must be a subclass for this now..

Reply:
OSError now includes IOError, since Python 3.3
basically OSError is an alias for IOError and other exceptions
they've been even removed from the hierarchy https://docs.python.org/3/library/exceptions.html#exception-hierarchy

Reply:
What I mean is, isn't it just `FileNotFoundError`?

Reply:
oh... yeah, let's be more precise